### PR TITLE
Convert diag_tables.php

### PR DESCRIPTION
**Pull request by sbeaver-netgate**

### CODE REVIEW

**Comment by SjonHortensius** [comment]
Can you apply the same changes I did in 4e74880 and #70? I'll complete the js if you'd like

**Comment by sbeaver-netgate** [comment]
OK

> On Apr 18, 2015, at 10:29 AM, SjonHortensius notifications@github.com wrote:
> 
> Can you apply the same changes I did in 4e74880 https://github.com/SjonHortensius/pfsense/commit/4e7488013ffe0ab161b732894b2a3474dcafc134 and #70 https://github.com/SjonHortensius/pfsense/pull/70? I'll complete the js if you'd like
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/SjonHortensius/pfsense/pull/87#issuecomment-94168107.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com mailto:sbeaver@netgate.com | www.netgate.com http://www.netgate.com/

**Comment by sbeaver-netgate** [comment]
Have you completed this conversion?

Your comment says "Apply same changes I did in . . . and #70, but #70 is also diag_routes.php

**Comment by SjonHortensius** [comment]
Obviously I meant the commit referenced there; c722847. Please apply the changes I made there to this file as well

**Comment by sbeaver-netgate** [comment]
You may be sure that had it been obvious to me I would not have asked for clarification :)

A new revision has just been sync’d. I look forward to your review.

> On Apr 20, 2015, at 1:01 PM, SjonHortensius notifications@github.com wrote:
> 
> Obviously I meant the commit referenced there; c722847 https://github.com/SjonHortensius/pfsense/commit/c722847eea2ea3601714fb72bb2b2af0d80f254e. Please apply the changes I made there to this file as well
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/SjonHortensius/pfsense/pull/87#issuecomment-94509496.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com mailto:sbeaver@netgate.com | www.netgate.com http://www.netgate.com/

File: usr/local/www/diag_tables.php [hunk 1]
```diff
@@ -119,59 +143,90 @@ function del_entry(entry) {
...
+
+		$section2->addInput(new Form_StaticText(
+			'Last update',
+			$last_updated
+		));
+
+		$section2->addInput(new Form_Button(
+			'Download',
+			'Download'
+		))->setHelp(gettext('Download the latest bogon data'))->removeClass('btn -primary')->addClass('btn-warning btn-sm');
```
Comment (SjonHortensius) [inline usr/local/www/diag_tables.php]: File looks good, but both this and the gettext call on line 214 aren't needed

Comment (sbeaver-netgate) [inline usr/local/www/diag_tables.php]: Removed as suggested.

Thanks.

> On Apr 23, 2015, at 10:52 AM, SjonHortensius notifications@github.com wrote:
> 
> In usr/local/www/diag_tables.php https://github.com/SjonHortensius/pfsense/pull/87#discussion_r28969455:
> 
> > -   ));
> >   +
> > -   $section2 = new Form_Section('Table Data');
> >   +
> > -   if($bogons) {
> > -       $last_updated = exec('/usr/bin/grep -i -m 1 -E "^# last updated" /etc/' . escapeshellarg($tablename) . '|cut -d"(" -f2|tr -d ")" ');
> >   +
> > -       $section2->addInput(new Form_StaticText(
> > -           'Last update',
> > -           $last_updated
> > -       ));
> >   +
> > -       $section2->addInput(new Form_Button(
> > -           'Download',
> > -           'Download'
> > -       ))->setHelp(gettext('Download the latest bogon data'))->removeClass('btn -primary')->addClass('btn-warning btn-sm');
> >   File looks good, but both this and the gettext call on line 214 aren't needed
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/SjonHortensius/pfsense/pull/87/files#r28969455.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com mailto:sbeaver@netgate.com | www.netgate.com http://www.netgate.com/

Comment (SjonHortensius) [inline usr/local/www/diag_tables.php]: @sbeaver-netgate It seems like you didn't commit or push this change

Comment (sbeaver-netgate) [inline usr/local/www/diag_tables.php]: I show no outstanding changes, but have sync’d my branch again.

> On Apr 25, 2015, at 4:18 AM, SjonHortensius notifications@github.com wrote:
> 
> In usr/local/www/diag_tables.php https://github.com/SjonHortensius/pfsense/pull/87#discussion_r29098927:
> 
> > -   ));
> >   +
> > -   $section2 = new Form_Section('Table Data');
> >   +
> > -   if($bogons) {
> > -       $last_updated = exec('/usr/bin/grep -i -m 1 -E "^# last updated" /etc/' . escapeshellarg($tablename) . '|cut -d"(" -f2|tr -d ")" ');
> >   +
> > -       $section2->addInput(new Form_StaticText(
> > -           'Last update',
> > -           $last_updated
> > -       ));
> >   +
> > -       $section2->addInput(new Form_Button(
> > -           'Download',
> > -           'Download'
> > -       ))->setHelp(gettext('Download the latest bogon data'))->removeClass('btn -primary')->addClass('btn-warning btn-sm');
> >   @sbeaver-netgate https://github.com/sbeaver-netgate It seems like you didn't commit or push this change
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/SjonHortensius/pfsense/pull/87/files#r29098927.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com mailto:sbeaver@netgate.com | www.netgate.com http://www.netgate.com/

**Pull request merged**